clients/cli/README.md

```diff
@@ -59,6 +60,75 @@ Before you can build and run this project, it is suggested you have the followin
     gcli --help
     ```
 
+## Table Commands
+
+The CLI supports the following operations for tables:
+
+### List Tables
+
+Lists all tables in a specified schema.
+
+```bash
+gcli table list --metalake <metalake_name> --name <catalog_name>.<schema_name>
+```
+
+Example:
+```bash
+gcli table list --metalake metalake_demo --name catalog_mysql.db
+```
+
+### Show Table Details
+
+Shows detailed information about a specific table.
+
+```bash
+gcli table details --metalake <metalake_name> --name <catalog_name>.<schema_name>.<table_name>
+```
+
+Example:
+```bash
+gcli table details --metalake metalake_demo --name catalog_mysql.db.iceberg_namespace_properties
+```
+
+### List Table Indexes
+
+Shows all indexes defined for a specific table.
+
+```bash
+gcli table details --metalake <metalake_name> --name <catalog_name>.<schema_name>.<table_name> --index
+```
+
+Example:
+```bash
+gcli table details --metalake metalake_demo --name catalog_mysql.db.iceberg_namespace_properties --index
+```
+
+### Create Table
+
+Creates a new table.
+
+```bash
+gcli table create --metalake <metalake_name> --name <catalog_name>.<schema_name>.<table_name>
+```
+
+Example:
+```bash
+gcli table create --metalake metalake_demo --name catalog_mysql.db.iceberg_namespace_properties
+```
+
+### Delete Table
+
+Deletes a specified table.
+
+```bash
+gcli table delete --metalake <metalake_name> --name <catalog_name>.<schema_name>.<table_name>
+```
+
+Example:
+```bash
+gcli table delete --metalake metalake_demo --name catalog_mysql.db.iceberg_namespace_properties
+```
+
```

Comment:
The README is for developers, not users, so the commands need to go in `docs/cli.md` not here.